studies.py

```diff
@@ -31,23 +47,48 @@ def objective_single(trial: optuna.Trial) -> float:
         direction="maximize",
     )
 
-    def objective_single_dynamic(trial: optuna.Trial) -> float:
-        category = trial.suggest_categorical("category", ["foo", "bar"])
-        if category == "foo":
-            return (trial.suggest_float("x1", 0, 10) - 2) ** 2
-        else:
-            return -((trial.suggest_float("x2", -10, 0) + 5) ** 2)
-
     study.optimize(objective_single_dynamic, n_trials=50)
-    studies.append(study)
+    studies[study.study_name] = study
 
     # No trials single-objective study
     optuna.create_study(study_name="A single objective study that has no trials", storage=storage)
     return studies
 
 
-def create_multi_objective_studies() -> List[Study]:
-    studies = []
+def create_multiple_single_objective_studies() -> Dict[str, StudiesType]:
```

Comment:
In the current implementation, we need to be careful to avoid conflicts with keys of dict objects returned by other create_xxx functions. Does it make sense to return `List[Tuple[str, StudiesType]]` instead of `Dict[str, StudiesType]`?

Reply:
Thank you for your comment. That sounds reasonable to me. I'll apply your suggestion.

Reply:
Fixed in https://github.com/optuna/optuna-visualization-regression-tests/pull/3/commits/e6fda9f265616ec2f68dbcfd9e7c11abe5c4f0c3.
IMO, `List[Tuple[str,` StudiesType]]` is a bit complicated and maybe we need refactoring (e.g., defining class to handle groups of `studies`) for further extension. But, we can work on it in a follow-up PR.